docs/docs/user/references/cli-commands.md

```diff
@@ -18,9 +18,10 @@ $ kpops [OPTIONS] COMMAND [ARGS]...
 * `clean`: Clean pipeline steps
 * `deploy`: Deploy pipeline steps
 * `destroy`: Destroy pipeline steps
-* `generate`: Enriches pipelines steps with defaults.
+* `generate`: Enrich pipeline steps with defaults.
+* `render`: In addition to generate, render final...
```

Comment:
fix: do not truncate

Reply:
@sujuka99 do you know if it's supposed to be like this? would be better to see the full help for each of the commands

Reply:
ok, thanks for checking. If it's outside our control then so be it

Reply:
See https://github.com/pallets/click/issues/486

Could you try putting one of the `help` strings in `short_help`?

Reply:
Great! I've added a separate `short_help` for the command now, which should suffice

Reply:
Ah, I was thinking about putting the whole `help` in `short_help`

Reply:
yeah, I got that. But then I thought short help is named as such for a reason, so I gave it a shorter string. The full help is then visible for the command `kpops manifest --help`. I did the same for `kpops generate` now and I think it looks nice and clean. What do you think?

Reply:
Ah got it now. I think I was mixing up the 2 descriptions in the docs. I agree, it looks quite nice.